src/Files.App/UserControls/Widgets/WidgetsListControl.xaml

```diff
@@ -159,53 +159,53 @@
 		</Style>
 	</UserControl.Resources>
 
-	<Border>
-		<Grid>
-			<ListView ItemsSource="{x:Bind ViewModel.Widgets, Mode=OneWay}" SelectionMode="None">
+	<Grid>
+		<ListView ItemsSource="{x:Bind ViewModel.Widgets, Mode=OneWay}" SelectionMode="None">
 
-				<!--  Remove Reveal Highlight Effect  -->
-				<ListView.ItemContainerStyle>
-					<Style TargetType="ListViewItem">
-						<!--  Spacing  -->
-						<Setter Property="Margin" Value="4" />
-						<Setter Property="Template">
-							<Setter.Value>
-								<ControlTemplate TargetType="ListViewItem">
-									<ListViewItemPresenter
-										x:Name="Root"
-										HorizontalContentAlignment="Stretch"
-										VerticalContentAlignment="Stretch" />
-								</ControlTemplate>
-							</Setter.Value>
-						</Setter>
-					</Style>
-				</ListView.ItemContainerStyle>
-				<ListView.ItemTemplate>
-					<DataTemplate x:DataType="vm:WidgetsListControlItemViewModel">
-						<controls:Expander
-							Padding="16,0,16,0"
+			<!--  Remove Reveal Highlight Effect  -->
+			<ListView.ItemContainerStyle>
```

Comment:
I'm pretty sure we can remove these style overrides since WinUI doesn't use revel highlight anymore.